integration/ng-add-standalone/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Why did we add these configs? (out of curiosity)

Reply:
After updating the test apps stopped compiling because they couldn't find tslib. I couldn't track it down and assumed it was some Bazel issue so I ended up setting importHelpers: false.